<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Semforización</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../data.qrc">
    <normaloff>:/icons/res/logo.svg</normaloff>:/icons/res/logo.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_map">
       <attribute name="icon">
        <iconset resource="../../data.qrc">
         <normaloff>:/icons/res/applications-internet.png</normaloff>:/icons/res/applications-internet.png</iconset>
       </attribute>
       <attribute name="title">
        <string>&amp;Mapa</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="MapView" name="graphicsViewMap">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>200</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>400</height>
            </size>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_events">
       <attribute name="icon">
        <iconset resource="../../data.qrc">
         <normaloff>:/icons/res/application-vnd.oasis.opendocument.spreadsheet.png</normaloff>:/icons/res/application-vnd.oasis.opendocument.spreadsheet.png</iconset>
       </attribute>
       <attribute name="title">
        <string>&amp;Eventos</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Eventos de muy alta prioridad y alta prioridad</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="margin">
             <number>1</number>
            </property>
            <item>
             <widget class="QTableViewHighPrio" name="tableHighPrio"/>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Eventos de baja prioridad y muy baja prioridad</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="margin">
             <number>1</number>
            </property>
            <item>
             <widget class="QTableViewLowPrio" name="tableLowPrio"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_conf">
       <attribute name="icon">
        <iconset resource="../../data.qrc">
         <normaloff>:/icons/res/configure.png</normaloff>:/icons/res/configure.png</iconset>
       </attribute>
       <attribute name="title">
        <string>&amp;Configuracion</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QToolBox" name="toolBox_configuracion_comandos">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="pageComs">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>786</width>
             <height>423</height>
            </rect>
           </property>
           <attribute name="label">
            <string>C&amp;omandos</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Cambio de programa&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Concentrador</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBoxModelQuery" name="comboBox_concentrador"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Zona:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboPgmZona"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>UC:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboPgmUc"/>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_prog_manual">
                  <property name="text">
                   <string>Programa Manual</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_prog_auto">
                  <property name="text">
                   <string>Programa automático</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pageCOs">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>786</width>
             <height>423</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Concentradores y &amp;Unidades de Control</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Concentradores&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushAddCO">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../../data.qrc">
                    <normaloff>:/icons/res/list-add.png</normaloff>:/icons/res/list-add.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTableCO" name="tableCO"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Unidades de Control&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QToolButton" name="toolAddUC">
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTableUC" name="tableUC"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSCADA">
    <property name="title">
     <string>&amp;SCADA</string>
    </property>
    <addaction name="actionUsuario"/>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Herramientas</string>
    </property>
    <widget class="QMenu" name="menuBarra_de_herramientas">
     <property name="title">
      <string>Barra de herramientas</string>
     </property>
     <addaction name="actionScada"/>
    </widget>
    <addaction name="actionSCADA_Config"/>
    <addaction name="actionLogs"/>
    <addaction name="actionDBUrl"/>
    <addaction name="menuBarra_de_herramientas"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>A&amp;yuda</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuAdmin">
    <property name="title">
     <string>&amp;Administración</string>
    </property>
    <addaction name="actionMapa"/>
   </widget>
   <addaction name="menuSCADA"/>
   <addaction name="menuAdmin"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="scadaToolbar">
   <property name="windowTitle">
    <string>SCADA</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
  </widget>
  <widget class="QToolBar" name="toolBar_mapa">
   <property name="windowTitle">
    <string>Edición de mapa</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMapa"/>
  </widget>
  <widget class="QToolBar" name="toolBar_mapa_visor">
   <property name="windowTitle">
    <string>Visor de mapa</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_debug">
   <property name="windowTitle">
    <string>Debug</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionIO"/>
   <addaction name="actionLimpiar_Eventos"/>
   <addaction name="actionSCADA_Config"/>
  </widget>
  <widget class="QToolBar" name="toolBar_impresion">
   <property name="windowTitle">
    <string>Impresión</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImprimir"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>Sobre...</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>Sobre Qt...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data.qrc">
     <normaloff>:/icons/res/player-time.png</normaloff>:/icons/res/player-time.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="statusTip">
    <string>Inicia el motor scada</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../../data.qrc">
     <normaloff>:/icons/res/process-stop.png</normaloff>:/icons/res/process-stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionSCADA_Config">
   <property name="icon">
    <iconset resource="../../data.qrc">
     <normaloff>:/icons/res/configure.png</normaloff>:/icons/res/configure.png</iconset>
   </property>
   <property name="text">
    <string>SCADA Config</string>
   </property>
  </action>
  <action name="actionLogs">
   <property name="text">
    <string>Logs</string>
   </property>
  </action>
  <action name="actionMapa">
   <property name="icon">
    <iconset resource="../../data.qrc">
     <normaloff>:/icons/res/applications-accessories.png</normaloff>:/icons/res/applications-accessories.png</iconset>
   </property>
   <property name="text">
    <string>Edición Mapa</string>
   </property>
  </action>
  <action name="actionEdici_n_Esquina">
   <property name="text">
    <string>Edición Esquina</string>
   </property>
  </action>
  <action name="actionUsuario">
   <property name="text">
    <string>Cambio de usuario</string>
   </property>
  </action>
  <action name="actionDBUrl">
   <property name="text">
    <string>Mostrar RFC1783 DB URL</string>
   </property>
  </action>
  <action name="actionScada">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scada</string>
   </property>
  </action>
  <action name="actionIO">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data.qrc">
     <normaloff>:/icons/res/strigi.png</normaloff>:/icons/res/strigi.png</iconset>
   </property>
   <property name="text">
    <string>DebugIO</string>
   </property>
  </action>
  <action name="actionLimpiar_Eventos">
   <property name="icon">
    <iconset resource="../../data.qrc">
     <normaloff>:/icons/res/draw-eraser.png</normaloff>:/icons/res/draw-eraser.png</iconset>
   </property>
   <property name="text">
    <string>Limpiar Eventos</string>
   </property>
  </action>
  <action name="actionImprimir">
   <property name="icon">
    <iconset resource="../../data.qrc">
     <normaloff>:/icons/res/printer.png</normaloff>:/icons/res/printer.png</iconset>
   </property>
   <property name="text">
    <string>Imprimir</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTableViewHighPrio</class>
   <extends>QTableView</extends>
   <header>dbview</header>
  </customwidget>
  <customwidget>
   <class>QTableViewLowPrio</class>
   <extends>QTableView</extends>
   <header>dbview</header>
  </customwidget>
  <customwidget>
   <class>QTableCO</class>
   <extends>QTableView</extends>
   <header>dbview</header>
  </customwidget>
  <customwidget>
   <class>QTableUC</class>
   <extends>QTableView</extends>
   <header>dbview</header>
  </customwidget>
  <customwidget>
   <class>MapView</class>
   <extends>QGraphicsView</extends>
   <header>mapview</header>
   <slots>
    <signal>zoomIn()</signal>
    <signal>zoomOut()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>QComboBoxModelQuery</class>
   <extends>QComboBox</extends>
   <header>widgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../data.qrc"/>
 </resources>
 <connections/>
</ui>
